app/src/main/java/pl/droidsonroids/droidsmap/feature/room/api/RoomImagesEndpoint.kt

```diff
@@ -0,0 +1,7 @@
+package pl.droidsonroids.droidsmap.feature.room.api
+
+import io.reactivex.Single
+
+interface RoomImagesEndpoint {
+    fun getRoomData(imageId : String): Single<String>
```

Comment:
Nie wiem dokładnie po co jest to room data, ale jeśli to możliwe to nazwałbym to bym to konkretniej.